config/configLoadHelpers_test.go

```diff
@@ -147,3 +151,133 @@ func Test_ConfigHashMetrics(t *testing.T) {
 		})
 	}
 }
+
+// creates temporary yaml files from the strings passed in and returns a slice of their filenames
+func createTempConfigs(t *testing.T, cfgs ...string) []string {
+	tmpDir, err := os.MkdirTemp("", "")
```

Comment:
we can use `t.TempDir()` here so that we can take advantage of the automatic cleanup after all tests have finished running

Reply:
Nice, I didn't know about that one!

Reply:
I also made this change in the other places we didn't use it. Thanks!